l10nzhtw.properties

```diff
@@ -0,0 +1,14 @@
+category=Localization
+description=SonarQube Traditional Chinese Pack
```

Comment:
Please update the description to mention incompatibility with the l10n-zh pack.

Reply:
I have revised compatibility in sqVersion and mention incompatibility with the l10n-zh pack. Please see if there is any problem. Thank you.